geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl

```diff
@@ -11,8 +11,9 @@ parser = argparse.ArgumentParser(description="Build the project,")
 parser.add_argument("--config", action="store_true", help="Build only the configuration image")
 parser.add_argument("--geoportal", action="store_true", help="Build only the geoportal image")
 parser.add_argument("--env", action="store_true", help="Force to create the `.env` file")
+parser.add_argument("--pgschema", default="main", help="The used main schema name in PostgreSQL")
```

Comment:
"The used main schema name in PostgreSQL" -> "The main schema name used in PostgreSQL"